benchmarks/portable/bench_harness/__init__.py

```
@@ -0,0 +1,21 @@
+    resolve_scenario,
+)
+
+__all__ = [
+    "Phase",
+    "PhaseType",
+    "PHASE_TINTS",
+    "parse_phase_spec",
+    "parse_duration",
+    "SCENARIOS",
+    "resolve_scenario",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)